ccb_vaultic/ccb_vaultic.c

```diff
@@ -0,0 +1,1021 @@
+/*
+ * ccb_vaultic.c
+ *
+ * Copyright (C) 2023 wolfSSL Inc.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/* System includes */
+#include <stdlib.h>  /* For NULL */
+#include <string.h>  /* For memset/cpy */
+
+/* wolfSSL configuration */
+#include "wolfssl/options.h"
```

Comment:
We usually do something like this to make sure the right build settings are pulled in for configure or custom build:

```
#ifndef WOLFSSL_USER_SETTINGS
    #include "wolfssl/options.h"
#endif
#include "wolfssl/wolfcrypt/settings.h"
```

Reply:
I'll update this.